es-ds-docs/pages/molecules/card.vue

```diff
@@ -0,0 +1,232 @@
+<template>
+    <div>
+        <h1>
+            Card
+        </h1>
+        <p class="mb-500">
+            Cards are used to visually group related information on a page. There are two variants:
+            display (default) and interactive.
+        </p>
+
+        <div class="mb-500">
+            <h2>
+                Display
+            </h2>
+            <p>
+                This is the default styling for a card.
+            </p>
+            <es-card>
+                <h3>
+                    Card title
+                </h3>
+                <p class="mb-0">
+                    Card contents
+                </p>
+            </es-card>
+        </div>
+
+        <div class="mb-500">
+            <h2>
+                Interactive (button)
+            </h2>
+            <p>
+                Use <code>variant="interactive"</code> to change the card's visual treatment to
+                indicate that it's clickable. By default, it will automatically become a button.
+            </p>
+            <es-card 
+                onclick="alert('The interactive card has been clicked.')"    
```

Comment:
For the sake of doing this the Vue way, can we put this inside the `@click` handler, where the modalVisible currently is?

Reply:
Since `alert` is a global and not a javascript function, it does not work in the `@click` handler. I wrapped the alert in a function called `notify` and updated the TODO.

Reply:
Oh interesting!